### ISSUE TITLE
Test the `MandVCurvefitEstimator` class

### ISSUE THREAD

**Issue by peterphung2043** [peterphung2043/mandvmodeling#5]
In [`test_sklearn_interop.py`](https://github.com/cunybpl/changepointmodel/blob/development/tests/test_sklearn_interop.py#L2) from `cunybpl/changepointmodel`, the docs explain that `CurvefitEstimator` is tested again `sklearn`'s model training API for interoperability. Since `MandVCurvefitEstimator` inherits from `CurvefitEstimator`, we should test to see if the same interoperability still applies for this child class.

Basically rewrite all of the tests in `test_sklearn_interop.py` with `MandVCurvefitEstimator` and apply the necessary assertions.

**Comment by peterphung2043** [comment]
`test_estimator_integration_daily.py` and `test_estimator.py` both test the `MandVCurvefitEstimator` class

Refer to this commit hash
234af894da14e787956f3101b77b602a2c3e45fa

**Comment by peterphung2043** [comment]
This commit (7e2eabcdff824faf67517e5d35bc1ee961aa5806) has a `test_sklearn_interop.py` unit test module that tests the `MandVCurvefitEstimator` class for interoperability with `sklearn`
